gitlab/client_repository_commit.go

```diff
@@ -64,18 +64,22 @@ func (c *CommitClient) listPage(ctx context.Context, branch string, perPage, pag
 }
 
 // Create creates a commit with the given specifications.
-func (c *CommitClient) Create(ctx context.Context, branch string, message string, files []gitprovider.CommitFile) (gitprovider.Commit, error) {
+func (c *CommitClient) Create(ctx context.Context, branch string, message string, files []gitprovider.File) (gitprovider.Commit, error) {
 
 	if len(files) == 0 {
 		return nil, fmt.Errorf("no files added")
 	}
 
-	commitActions := make([]*gitlab.CommitAction, 0)
+	fileAction := gitlab.FileCreate
+
+	commitActions := make([]*gitlab.CommitActionOptions, 0)
 	for _, file := range files {
-		commitActions = append(commitActions, &gitlab.CommitAction{
-			Action:   gitlab.FileCreate,
-			FilePath: *file.Path,
-			Content:  *file.Content,
+		filePath := *file.Path
+		fileContent := *file.Content
+		commitActions = append(commitActions, &gitlab.CommitActionOptions{
+			Action:   &fileAction,
+			FilePath: &filePath,
+			Content:  &fileContent,
```

Comment:
This change seems unmotivated by the description of the PR.
Is it caused by the version change?

Reply:
I'll add a description for this